src/test/java/com/google/api/codegen/util/ProtoParserTest.java

```diff
@@ -201,6 +201,24 @@ public void testGetMethodSignatures() {
         .containsExactly("name", "message", "string_builder");
   }
 
+  @Test
+  public void testEmptySignature() {
+    // Test that we can detect empty method signatures.
```

Comment:
Can we also test the case when there is no method signature? (or is that already tested?)

Reply:
Sure! Done.